homeassistant/components/generic/config_flow.py

```diff
@@ -261,15 +268,37 @@ async def async_test_stream(
     return {}
 
 
+def register_preview(hass: HomeAssistant, flow_id: str, user_input: dict[str, Any]):
+    """Set up previews for camera feeds during config flow."""
+    hass.data.setdefault(DOMAIN, {})
+
+    if not hass.data[DOMAIN].get(PREVIEWS):
+        _LOGGER.debug("Registering camera image preview handler")
+        hass.http.register_view(CameraImagePreview(hass))
+    hass.data[DOMAIN][PREVIEWS] = {}
```

Comment:
Can you remind me: Is this alright that it wipes everything or is it worth preserving other flow ids? (e.g. `hass.data[DOMAIN].setdefault(PREVIEWS, {})`

Reply:
This would only be an issue in the very unlikely case of two different users configuring cameras simultaneously, which is something I proposed not supporting early on.

But yes, in this latest arrangement it could probably be allowed as you suggest, as long as we reliably cleared up old data somehow.

The ={} clears up any old data.

Reply:
@allenporter Now no longer an issue with the other changes.